### ISSUE TITLE
Expose the import order setting in the format preview page

### ISSUE THREAD

**Issue by jdneo** [microsoft/vscode-java-pack#1032]
We can see many users are bothered by the import order things, like:

https://stackoverflow.com/questions/73139834/vscode-keeps-changing-sort-order-of-import-when-saving-changes/73146433#73146433

@CsCherrYY  Does it make sense if we expose the import order setting in the format preview page? How does other IDEs handle that?

**Comment by CsCherrYY** [comment]
Actually the format settings only affect the blank line between imports. I guess the user might set the configuration `editor.codeActionsOnSave` so the organize imports is triggered when he saves the file.

**Comment by jdneo** [comment]
Yes, I mean it's not limited to only show content in the xml file. Like the indentation section we displayed now, it connects with the VS Code settings.

And in other IDEs, the import order is part of the sections in the format settings:

![image](https://user-images.githubusercontent.com/6193897/181409704-1fbd88c4-e776-42fd-a860-e35b1d5a0469.png)
